server/middleware/grpc-client/format-response/format-history-event-details/format-external-workflow-execution-signaled-event-attributes.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2022 Uber Technologies Inc.
+//
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+const atob = require('atob');
+
+const formatExternalWorkflowExecutionSignaledEventAttributes = ({
+  control,
+  initiatedEventId,
+  ...eventAttributes
+}) => ({
+  ...eventAttributes,
+  control: control ? parseInt(atob(control)) : null,
```

Comment:
In general `control` it is a byte array. Why is it being parsed as int?

Reply:
Let me review this one again. I recall I made this change to match what TChannel was returning but I'll try to find another workflow in staging to compare again and tag you. Thanks!

Reply:
It may be a case that we only call atob on control and return that instead of parsing to integer. I'll review today and let you know.

Reply:
talked offline about this and agreed to make a change only if we need to during testing in staging.